### DISCUSSION TITLE
contributing/development/compiling/getting_source

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#229] [User-contributed notes]
# contributing/development/compiling/getting_source

Downloading the Godot source code: Before getting into the SCons build system and compiling Godot, you need to actually download the Godot source code. The source code is available on GitHub and wh...

https://docs.godotengine.org/en/latest/contributing/development/compiling/getting_source.html

**Comment by jinyangcruise** [comment]
Can someone add the content about how to get the source of a Dev snapshot like `Godot 4.4 dev 3`?

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot-docs/pull/10201
